app.js

```diff
@@ -100,6 +105,66 @@ router.post('/account', ratelimit({
     ctx.body = await masterAccount.createAccount(newAccountId, newAccountPublicKey, NEW_ACCOUNT_AMOUNT);
 });
 
+// TODO: Adjust gas to correct amounts
+const MAX_GAS_FOR_ACCOUNT_CREATE = process.env.MAX_GAS_FOR_ACCOUNT_CREATE || '100000000000000';
```

Comment:
nit: seems like good occasion to move account creation into separate module in `middleware/`

Reply:
Done!